tasks/libs/ciproviders/github_api.py

```diff
@@ -64,11 +64,13 @@ def get_branch(self, branch_name):
                 return None
             raise e
 
-    def create_pr(self, pr_title, pr_body, base_branch, target_branch):
+    def create_pr(self, pr_title, pr_body, base_branch, target_branch, draft=False):
```

Comment:
was necessary to create a draft PR. I wanted to use a draft PR for this because collector dependency upgrade can break existing tests/codepaths and I want to be able to fix them before marking the PR ready.